pkg/core/resources/store/error.go

```diff
@@ -0,0 +1,61 @@
+package store
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+var (
+	ErrInvalidOffset   = errors.New("invalid offset")
+	ErrIsAlreadyExists = errors.New("already exists")
+	ErrConflict        = errors.New("conflict")
+	ErrNotFound        = errors.New("not found")
+	ErrInvalid         = errors.New("invalid")
+	ErrBadRequest      = errors.New("bad request")
```

Comment:
Some of these errors are too specific to have their own type, yet they are too generic as well as you can't pass in a reason. For example ErrInvalid can never have a reason so the errors are useless to the end consumer of the api.